_posts/contests/2021-02-05-Performance-analysis-and-tuning-contest-4.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: Performance analysis and tuning contest &#45&#51.
+categories: contest
+---
+
+**Contents:**
+* TOC
+{:toc}
+
+Welcome to the 4th edition of our performance analysis and tuning contest. If you see this post and haven't read my initial post about the [contest]({{ site.url }}/blog/2019/02/02/Performance-optimization-contest), I encourage you to read it first. 
+
+The fourth edition of the contest will be ran by Ivica Bogosavljevic from [Johny's Software Lab](https://johnysswlab.com) blog. Ivica also writes about software performance, so feel free to go and check out his blog, there is a ton of useful content there.
+
+------
+**Subscribe to my [mailing list](https://mailchi.mp/4eb73720aafe/easyperf), support me on [Patreon](https://www.patreon.com/dendibakh) or by PayPal [donation](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=TBM3NW8TKTT34&currency_code=USD&source=url).**
+
+------
+
+Guest writer: [Ivica Bogosavljevic](https://johnysswlab.com/author/ibogi/)
+
+The benchmark for the 4th edition is canny edge detection algorithm. The source code, compilation and run script as well as test image is available in the [code repository](TODO: Add link to the repository).
```

Comment:
The link would be:
`https://github.com/dendibakh/dendibakh.github.io/tree/master/_posts/code/PerfContest/4`